src/features/contact/contactverzoek/formulier/ContactverzoekFormulier.vue

```diff
@@ -472,21 +471,55 @@ watch(
 
 //////////////////////////////////////////////////////
 
+const hasContact = computed(
+  () =>
+    !!form.value.telefoonnummer1 ||
+    !!form.value.telefoonnummer2 ||
+    !!form.value.emailadres,
+);
+
+const noContactMessage =
+  "Vul minimaal een telefoonnummer of een e-mailadres in";
+
 watch(
-  [
-    telEl,
-    () =>
-      !!form.value.telefoonnummer1 ||
-      !!form.value.telefoonnummer2 ||
-      !!form.value.emailadres,
-  ],
-  ([el, hasContact]) => {
-    if (!el) return;
+  [telEl, hasContact],
+  ([el, bool]) => el && el.setCustomValidity(!bool ? noContactMessage : ""),
+);
+
+// klantinteracties/api/v1/schema/#tag/digitale-adressen/operation/digitaleadressenCreate
+const TELEFOON_PATTERN =
```

Comment:
een comment met een verwijzing naar de bron is hier ook wel nuttig, net als bij email

Reply:
bron opgezocht en als comment toegevoegd